Packs/CortexXDR/ReleaseNotes/6_1_41.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Fixed an issue where the *exclude_fields* parameter would fail upon receiving a None value.
```

Comment:
Please change this to be more user-facing (i.e. parameter display name + and when not providing a value).